components/chassis.py

```diff
@@ -94,15 +92,11 @@ def get_speed(self):
     def set(self, desired_state: SwerveModuleState):
 
         if abs(desired_state.speed) < 1e-3:
-            if abs(self.last_speed) > 1e-13:
-                self.drive.setIntegralAccumulator(0, 0, 0)
-                self.drive.set(ctre.ControlMode.Velocity, 0)
-                self.steer.set(ctre.ControlMode.Velocity, 0)
-                self.last_speed = 0
+            self.drive.set(ctre.ControlMode.Velocity, 0)
+            self.steer.set(ctre.ControlMode.Velocity, 0)
```

Comment:
I do wonder if setting steer to velocity of 0 is a good idea if we normally run it on position PID...

Reply:
I dont think it really matters for now since things are working fine. This is the easiest fix for a pretty loud bug lmao

Reply:
Well, if you say it works fine, then :rocket: